apps/archive/archive.py

```diff
@@ -599,7 +599,7 @@ def on_replaced(self, document, original):
             subject=get_subject(original),
         )
         push_content_notification([document, original])
-        self.cropService.update_media_references(document, original)
+        await self.cropService.update_media_references(document, original)
```

Comment:
[on_replaced](https://github.com/superdesk/superdesk-core/blob/465629528fca523bd9000bb79bc12863f86fa20b/apps/archive/archive.py#L591) needs to be changed to an `async` method, as this is causing the code not to run (and tests)